reactor-core/src/test/java/reactor/core/publisher/MonoCreateTest.java

```diff
@@ -298,6 +298,19 @@ public void sinkApiErrorAfterErrorBubblesAndDrops() {
 		            .hasOperatorErrorWithMessage("boom2");
 	}
 
+	@Test
+	public void delayUntilTriggerProviderThrows() {
+		Mono<String> triggerProviderThrows = Mono.<String>create(sink ->
+			sink.success("foo")
+		)
+		.delayUntil(str -> {
```

Comment:
according to Reactor's codestyle (see https://github.com/reactor/reactor-core/tree/master/codequality), such method calls should be indented. Could you please apply it?